ci-operator/step-registry/shiftstack/aliveness/shiftstack-aliveness-commands.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+set -Eeuo pipefail
+
+export OS_CLOUD
+export OS_CLIENT_CONFIG_FILE="${SHARED_DIR}/clouds.yaml"
+CLUSTER_NAME=$(<"${SHARED_DIR}/CLUSTER_NAME")
+# Recycling BASTION_FLAVOR as it's a small flavor we can re-use.
+TESTING_FLAVOR="${TESTING_FLAVOR:-$(<"${SHARED_DIR}/BASTION_FLAVOR")}"
+
+if openstack keypair show shiftstack-ci > /dev/null 2>&1; then
+  echo "DEBUG: Keypair was found for 'shiftstack-ci'; removing it.."
+  openstack keypair delete shiftstack-ci
+fi
+openstack keypair create --public-key ${CLUSTER_PROFILE_DIR}/ssh-publickey shiftstack-ci >/dev/null
+eval "$(ssh-agent)"
+if ! whoami &> /dev/null; then
+  if [ -w /etc/passwd ]; then
+    echo "${IMAGE_USER:-centos}:x:$(id -u):0:${IMAGE_USER:-centos} user:${HOME}:/sbin/nologin" >> /etc/passwd
+  fi
+fi
+ssh-add "${CLUSTER_PROFILE_DIR}/ssh-privatekey"
+
+set +e
+echo "DEBUG: Running aliveness check script..."
+./server.sh -d -t -u ${IMAGE_USER} -f ${TESTING_FLAVOR} -i ${TESTING_IMAGE} -e ${OPENSTACK_EXTERNAL_NETWORK} -k shiftstack-ci shiftstack-ci-${CLUSTER_NAME}
```

Comment:
I'm having trouble understanding why this job is needed. If the cloud is unhealthy, wouldn't we know it from the other jobs' results? If the failure isn't clear, then perhaps we should instead make the failure to provision resource more obvious in our jobs, or in openshift directly.

Reply:
the periodic jobs run every 3 days and report on slack.
presubmits jobs don't report on slack, so it makes it more complicated to figure out if something is going wrong in the cloud.